modules/op-setting-annotations-labels-namespace.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-setting-annotations-labels-namespace_{context}"]
+= Setting labels and annotations for the {pipelines-shortname} installation namespace
+
+You can set labels and annotations on the `openshift-pipelines` namespace in which the operator installs {pipelines-shortname}.
+
+[NOTE]
+====
+Changing the name of the `openshift-pipelines` namespace is not supported at present
+====
+
+To set the labels and annotations, use the `spec.targetNamespaceMetadata` specification in the `TektonConfig` custom resource (CR).
+
+.Example of setting labels and annotations for the `openshift-pipelines` namespace
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  targetNamespaceMetadata:
+    labels: {"example-label":"example-value"}
+    annotations: {"example-annotation":"example-value"}
```

Comment:
@xenolinux this would be a syntax definition while what I have is an example. The reason I went for an example is because the surrounding similar modules are examples. For example (yeah), https://75535--ocpdocs-pr.netlify.app/openshift-pipelines/latest/install_config/customizing-configurations-in-the-tektonconfig-cr.html#op-changing-default-service-account_customizing-configurations-in-the-tektonconfig-cr has "stable", not "<api_fields_level>"

Reply:
After looking at other sections, I agree with you. +1 to keep consistency with other sections.